README.md

```diff
@@ -1,42 +1,39 @@
 # ROCm for Arch Linux
 This repository hosts a collection of [Arch Linux](https://www.archlinux.org/) [PKGBUILDs](https://wiki.archlinux.org/index.php/PKGBUILD) for the [AMD ROCm Platform](https://rocmdocs.amd.com/en/latest/).
-These scripts implement a great portion of the stack, ranging from low level interfaces, over compilers and high-level application librariers.
+These scripts implement a great portion of the stack, ranging from low-level interfaces, over compilers to high-level application librariers.
 
 ## Installation
 The Arch Linux packages for ROCm are available on the [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository).
-Since many packages will be installed, it is recommended to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers)
-like [`yay`](https://aur.archlinux.org/packages/yay/).
-
-It is also recommended to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will
-greatly speed up your installation time.
+Since many packages will be installed, we recommend to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers) like [`yay`](https://aur.archlinux.org/packages/yay/).
+Binary versions are hosted by [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu). Using them will greatly speed up your installation time.
```

Comment:
Let's keep the same wording as before since in this version we are not recommending arch4edu.

Reply:
```suggestion
Since many packages will be installed, it is recommended to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers)
like [`yay`](https://aur.archlinux.org/packages/yay/).

It is also recommended to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will
greatly speed up your installation time.
```